lib/resources/auditd.rb

```diff
@@ -30,11 +30,26 @@ class AuditDaemon < Inspec.resource(1)
     "
 
     def initialize
-      @content = inspec.command('/sbin/auditctl -l').stdout.chomp
+      unless inspec.command('/sbin/auditctl').exist?
+        raise Inspec::Exceptions::ResourceFailed,
+              'Command `/sbin/auditctl` does not exist'
+      end
+
+      auditctl_cmd = '/sbin/auditctl -l'
+      result = inspec.command(auditctl_cmd)
+
+      if result.exit_status != 0
+        raise Inspec::Exceptions::ResourceFailed,
+              "Command `#{auditctl_cmd}` failed with error: #{result.stderr}"
```

Comment:
Did we want to pass up a note to the user here that they cannot use this resources without escalated privileges? ( e.g. root or sudo )

Reply:
I originally had thought that, but then I found out that the stderr when that is the case is informative enough. 

```
[vagrant@localhost ~]$ auditctl -l
You must be root to run this program.
```